src/components/ArticleList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Box, Grommet } from "grommet";
+import { grommet } from "grommet/themes";
+import axios from "axios";
+import { connect } from "react-redux";
+
+class ArticleList extends Component {
+  componentDidMount() {
+    axios.get("/articles").then(response => {
+      this.props.dispatch({
+        type: "ARTICLES",
+        payload: { articleList: response.data.articles }
+      });
+    });
+  }
+  render() {
+    let articleDisplay;
+    if (this.props.articleList !== []) {
+      articleDisplay = this.props.articleList.map(article => {
+        return (
+          <>
+            <Grommet full theme={grommet}>
```

Comment:
move this grommet tag down to return instead, now we're rendering one for each article. We don't need to do that.

Reply:
Done